runhouse/servers/obj_store.py

```diff
@@ -198,6 +201,16 @@ async def ainitialize(
                 "Warning, cluster servlet is not initialized. Object Store operations will not work."
             )
 
+        owner_has_den_token = Path(CONFIG_YAML_PATH).expanduser().exists()
```

Comment:
name here is a little confusing, really this is just checking for the config file, not the token itself that should be in it